### PR TITLE
ports tg's filter debug menu & minor filter refactor

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

ports tgstation/tgstation#55246

image

## Changelog
:cl:actionninja
add: New filter editor for admins + development purposes
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
horray now admins can fuck up even more things

File: code/game/atoms.dm [hunk 1]
```diff
@@ -924,6 +924,9 @@
 	VV_DROPDOWN_OPTION(VV_HK_ADD_REAGENT, "Add Reagent")
 	VV_DROPDOWN_OPTION(VV_HK_TRIGGER_EMP, "EMP Pulse")
 	VV_DROPDOWN_OPTION(VV_HK_TRIGGER_EXPLOSION, "Explosion")
+	VV_DROPDOWN_OPTION(VV_HK_RADIATE, "Radiate")
```
Comment (bloons3) [inline code/game/atoms.dm]: does this have a side effect?

can't find code for this in this PR

**Review by bloons3 — CHANGES_REQUESTED** [review]

**Comment by bloons3** [comment]
no don't remove it I wanna adminbus with it reeeee

https://github.com/tgstation/tgstation/blob/46acdbd52d98ee79fb4b823b5c09676756c0d8e4/code/game/atoms.dm#L1248

**Comment by jupyterkat** [comment]
like you said, it does literally nothing because I literally didn’t bring any code from it to here. wait for a separate pr

**Comment by bloons3** [comment]
😿

**Review by utilbird — APPROVED** [review]

**Pull request merged**